src/services/services.ts

```diff
@@ -1807,6 +1807,15 @@ namespace ts {
             return range && createTextSpanFromRange(range);
         }
 
+        function getSpanForPosition(fileName: string, position: number): TextSpan {
+            synchronizeHostData();
+
+            const sourceFile = getValidSourceFile(fileName);
+            const node = getTouchingPropertyName(sourceFile, position, /*includeJsDocCcomment*/ false);
+

```

Comment:
what if this position is within a comment? do you want to verify that it is at least a token? identifier? can you elaborate on the scenario here?

Reply:
Added JsDoc for comments, based on our discussion I had changed the design of the command. This is no longer valid.